src/quo/components/list_items/account_list_card/schema.cljs

```diff
@@ -1,19 +1,19 @@
 (ns quo.components.list-items.account-list-card.schema)
 
 (def ^:private ?base
-  [:map
-   [:action {:optional true} [:enum :icon :none]]
-   [:blur? {:optional true} [:maybe :boolean]]
-   [:on-press {:optional true} [:maybe fn?]]
-   [:account-props
+  [:schema.common/map {:optional true}
```

Comment:
FYI: To demonstrate the flexibility, this is an example where `:maybe` is not used, but `:optional` is.